projects/addon-charts/components/arc-chart/arc-chart.component.ts

```diff
@@ -0,0 +1,107 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    HostBinding,
+    Inject,
+    Input,
+} from '@angular/core';
+import {DomSanitizer, SafeValue} from '@angular/platform-browser';
+import {tuiDefaultProp} from '@taiga-ui/cdk';
+import {TuiSizeXL} from '@taiga-ui/core';
+
+// 3/4 with 1% safety offset
+const ARC = 0.76;
+
+const SIZE: Record<TuiSizeXL, number> = {
+    m: 9,
+    l: 11,
+    xl: 16,
+};
+
+const WIDTH: Record<TuiSizeXL, number> = {
+    m: 0.25,
+    l: 0.375,
+    xl: 0.5625,
+};
+
+const GAP: Record<TuiSizeXL, number> = {
+    m: 0.125,
+    l: 0.1875,
+    xl: 0.25,
+};
```

Comment:
maybe, move to another file `arc-chart.options.ts`?

Reply:
I don't want to add noise to exported entities.